x/stakeibc/keeper/deposit_records.go

```diff
@@ -66,8 +66,8 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 		}
 		delegateAddress := delegateAccount.GetAddress()
 
-		blockHeight, found := k.GetLightClientHeightSafely(ctx, hostZone.ConnectionId)
-		if !found {
+		blockHeight, err := k.GetLightClientHeightSafely(ctx, hostZone.ConnectionId)
```

Comment:
Given tendermint comes to consensus on time (`BlockTime` should be the same across chains), I wonder if we should move away from this pattern entirely

Reply:
One issue here is that the LC staleness makes the timeouts shorter

Reply:
resolved here https://github.com/Stride-Labs/stride/pull/245